beta_dist_plot.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        latexify(width_scale_factor=2, fig_height=1.5)
</pre><p>Let us set height=2 as per the book and then increase legend size accordingly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done